runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -159,8 +159,8 @@
 	private int lastAnimation;
 	private boolean widgetHiddenChangedOnPvpWorld;
 	private ElapsedTimer tzhaarTimer;
-	private int imbuedHeartClickTick = -1;
-	private int lastBoostedMagicLevel = -1;
+	private TimerTimer imbuedHeartTimer;
+	private int lastImbuedHeartVarb = -1;
```

Comment:
It doesn't look like this needs to default to `-1` because `0` is a no-op. (doesn't add a timer)